conf/lookup/BCD-Port.csv

```diff
@@ -15520,4 +15518,101 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 100515,ZM,3,ZM2,,Private dock in Zambia,Doc preifat yng Zambia,0,4,NULL,0,
 100516,ZW,3,ZW2,,Private dock in Zimbabwe,Doc preifat yng Zimbabwe,0,4,NULL,0,
 100517,FI,2,HEL,,Helsinki Vantaa,Helsinki Vantaa,0,3,NULL,0,1
-100518,ES,3,ES2,,Santander Ferry Port,Santander Ferry Port,0,3,NULL,0,1
\ No newline at end of file
+100518,ES,3,ES2,,Santander Ferry Port,Santander Ferry Port,0,3,NULL,0,1
+200000,UK,3,ARD,,Ardrossan - ARD,Ardrossan - ARD,,1,NULL,0,
+200001,UK,3,ASD,,Ashford - ASD,Ashford - ASD,,1,NULL,0,
+200002,UK,3,AVO,Portishead Royal Portbury Dock,Avonmouth - AVO,Avonmouth - AVO,,1,NULL,0,
+200003,UK,3,AYR,Girvan,Ayr - AYR,Ayr - AYR,,1,NULL,0,
+200004,UK,3,BAR,Millom,Barrow-In-Furness - BAR,Barrow-In-Furness - BAR,,1,NULL,0,
+200005,UK,3,BCC,,Beccles Airport - BCC,Beccles Airport - BCC,,1,NULL,0,
+200006,UK,3,BLY,Amble Berwick,Blyth - BLY,Blyth - BLY,,1,NULL,0,
+200007,UK,2,BOH,,Bournemouth (Hurn) Airport - BOH,Bournemouth (Hurn) Airport - BOH,,1,NULL,0,
+200008,UK,3,BOS,,Boston - BOS,Boston - BOS,,1,NULL,0,
+200009,UK,3,BWK,,Newcastle (North Sheilds Dock) - BWK,Newcastle (North Sheilds Dock) - BWK,,1,NULL,0,

```

Comment:
Not needed, everything included here:

33310,UK,3,TYN,Newcastle North Shields South Shields,Tyne Seaport - TYN,Tyne Seaport - TYN,,1,NULL,1,

Reply:
Separating out as 2 sides of the tyne, correcting spelling instead